### PR TITLE
6.1.2 and windows builds :)

### PR DESCRIPTION
**Pull request by djsutherland**
Might have to hit restart on some of these depending on CI timing, but maybe it'll work while I'm commuting in.

cc @vigsterkr

### CODE REVIEW

**Comment by djsutherland** [comment]
hmm, can't rerender to have an appveyor build until there's a Windows build up. meh.

**Comment by djsutherland** [comment]
@vigsterkr Windows build of the Python interface isn't working (now that I have it actually running; ignore the Linux/Mac Xs here, I just canceled them). Possibly related to https://github.com/conda-forge/shogun-cpp-feedstock/issues/14 though I don't totally understand the output.

https://ci.appveyor.com/project/conda-forge/shogun-feedstock/build/1.0.3/job/dt613a4oqqnr2pey#L677

File: recipe/meta.yaml [hunk 1]
```diff
@@ -26,89 +27,114 @@ source:
...
+    - numpy 1.8.*   # [unix or (win and py27)]
+    - numpy 1.9.*   # [win and py35]
+    - numpy 1.11.*  # [win and py36]
     - shogun-cpp {{ version }}
-    - arpack 3.5.*
+    - arpack 3.5.*              # [not win]
     - bzip2 1.0.*
     - curl >=7.44.0,<8
-    - eigen
+    - eigen                     # [not win]
```
Comment (vigsterkr) [inline recipe/meta.yaml]: no need for this anymore... just use the latest eigen ;)

File: recipe/meta.yaml [hunk 2]
```diff
@@ -26,89 +27,114 @@ source:
...
+        -DENABLE_TESTING=OFF ^                       # [win]
+        -DENABLE_COVERAGE=OFF ^                      # [win]
+        -DUSE_SVMLIGHT=OFF ^                         # [win]
+        -DSWIG_EXECUTABLE=%LIBRARY_BIN%\\swig.exe ^  # [win]
+        -DLIBSHOGUN=OFF ^                            # [win]
+        -DPYTHON_EXECUTABLE=%PYTHON% ^               # [win]
+        -DINTERFACE_PYTHON=ON                        # [win]
+    if errorlevel 1 exit 1                           # [win]
+
+    msbuild /p:Configuration=Release shogun.sln        # [win]
```
Comment (vigsterkr) [inline recipe/meta.yaml]: use
```
cmake --build . --target _interface_python --config Release
```

**Comment by djsutherland** [comment]
Going to need to hit rebuild in a while once `shogun-cpp` actually builds and puts a new release up....

**Comment by vigsterkr** [comment]
yeeey 💃

**Comment by djsutherland** [comment]
AppVeyor passed :grin:

No substantial changes to the Mac recipe since the last version that passed on Travis, but going to check locally before merging.

**Comment by djsutherland** [comment]
Passed locally, merging. 🙌

**Pull request merged**